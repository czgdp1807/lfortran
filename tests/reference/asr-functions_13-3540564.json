{
    "basename": "asr-functions_13-3540564",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/functions_13.f90",
    "infile_hash": "f1507bfacd9c297f3b134f9251148a9a76bbe5e0ca648d085aa46687",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-functions_13-3540564.stdout",
    "stdout_hash": "1e463a72125bd56c822762120c05a74f9e5dd58bf2f6b808c811e11a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}