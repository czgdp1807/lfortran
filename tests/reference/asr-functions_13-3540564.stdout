(TranslationUnit 
    (SymbolTable
        1
        {
            iso_fortran_env:
                (IntrinsicModule lfortran_intrinsic_iso_fortran_env), 
            lfortran_intrinsic_builtin:
                (IntrinsicModule lfortran_intrinsic_builtin), 
            lfortran_intrinsic_string:
                (IntrinsicModule lfortran_intrinsic_string), 
            test_scan:
                (Program 
                    (SymbolTable
                        2
                        {
                            scan_kind4:
                                (ExternalSymbol 
                                    2 
                                    scan_kind4 
                                    4 scan_kind4 
                                    lfortran_intrinsic_string 
                                    [] 
                                    scan_kind4 
                                    Private
                                ), 
                            verify_kind4:
                                (ExternalSymbol 
                                    2 
                                    verify_kind4 
                                    4 verify_kind4 
                                    lfortran_intrinsic_string 
                                    [] 
                                    verify_kind4 
                                    Private
                                )
                            
                        }) 
                    test_scan 
                    [] 
                    [(Print 
                        () 
                        [(FunctionCall 
                            2 scan_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "ao" 
                                (Character 1 2 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 scan_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "ao" 
                                (Character 1 2 () [])
                            ))
                            ((LogicalConstant 
                                .true. 
                                (Logical 4 [])
                            ))] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 scan_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "c++" 
                                (Character 1 3 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 verify_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "ao" 
                                (Character 1 2 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 verify_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "foo" 
                                (Character 1 3 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 verify_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "c++" 
                                (Character 1 3 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 verify_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "c++" 
                                (Character 1 3 () [])
                            ))
                            ((LogicalConstant 
                                .true. 
                                (Logical 4 [])
                            ))] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )
                    (Print 
                        () 
                        [(FunctionCall 
                            2 verify_kind4 
                            () 
                            [((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            ((StringConstant 
                                "fortran" 
                                (Character 1 7 () [])
                            ))
                            (())] 
                            (Integer 4 []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )]
                )
            
        }) 
    []
)
